tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -136,6 +136,11 @@ func bindCmdlineFlags(cfg *config.Config, cmd *cobra.Command) {
 		&cfg.NetworkNamespace)
 
 	flag.BindEnv(fs, constants.CNIMode, "", "Whether to run as CNI plugin.", &cfg.CNIMode)
+
+	flag.BindEnv(fs, constants.SkipCleanup, "", "Skip cleanup of pre-existing and incompatible iptables rules", &cfg.SkipCleanup)
```

Comment:
```suggestion
	flag.BindEnv(fs, constants.NoReconcile, "", "Skip reconciliation of pre-existing and incompatible iptables rules, failing instead if drift detected", &cfg.NoReconcile)
```

cleanup != reconciliation so better to use different names.